### PR TITLE
chore: removes EOL Node.js version 8

### PR DESCRIPTION
**Pull request by helio-frota**

### CODE REVIEW

**Comment by gastaldi** [comment]
[test]

**Comment by quintesse** [comment]
@helio-frota don´t forget to update the metadata too! https://github.com/fabric8-launcher/launcher-booster-catalog/blob/master/metadata.json#L157-L176

**Comment by helio-frota** [comment]
@quintesse Thanks for the review!
I'll update these soon :+1:

**Comment by helio-frota** [comment]
@quintesse updated, please let me know if I miss something.

File: metadata.json [hunk 1]
```diff
@@ -153,28 +153,7 @@
       "metadata": {
         "pipelinePlatform": "node"
       },
-      "versions": [
-        {
-          "id": "v8-community",
-          "name": "8.x (Community)"
-        },
-        {
-          "id": "v10-community",
```
Comment (quintesse) [inline metadata.json]: @helio-frota I think you removed too much here, right? You probably should only have removed the v8 ones?

Comment (helio-frota) [reply metadata.json]: Thanks again. My mistake due distraction. Fixed :+1:

**Comment by gastaldi** [comment]
[test]

**Comment by quintesse** [comment]
Merged, thanks @helio-frota !

**Pull request merged**